Example/Tests/AppCommunicatorTests.swift

```diff
@@ -17,27 +17,33 @@ class AppCommunicatorTests: XCTestCase {
             return URL(string: "digime-ca-appID://\(testAction)")!
     }
     
+    var sut: DMEAppCommunicator!
+
+    override func setUp() {
+        sut = DMEAppCommunicator.shared()
```

Comment:
Please use a new instance of `DMEAppCommunicator` for each test rather than the shared instance i.e.
```swift
sut = DMEAppCommunicator()
```

If you missed it in the previous comment, this is because:
- it prevents interference from other tests (for instance a test in another file may set a callback handler on the shared instance thus affecting our tests)
- the app communicator is in the same consistent state before we start testing.  (For instance the `testAddingDMEAppCallback` test adds a callback handler to the shared instance).  This means that other tests using the shared instance could include that callback handler potentially giving misleading results.

Reply:
Looks like we can't do this as `init()` method has been made unavailable